_api-reference/cat/cat-allocation.md

```diff
@@ -36,7 +36,7 @@ include_deprecated: false
 -->
 ## Query parameters
 
-The following table lists the available query parameters. All query parameters are optional.
+The following table lists the available query parameters.
```

Comment:
@ReveristRealm Why did the "All query parameters are optional." get deleted? It is important information.